pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -160,6 +160,11 @@ type CertificateSpec struct {
 	// Options to control private keys used for the Certificate.
 	// +optional
 	PrivateKey *CertificatePrivateKey `json:"privateKey,omitempty"`
+
+	// EncodeUsagesInRequest controls whether key usages should be present
```

Comment:
This needs to be backported to: v1alph2, alpha3 and beta1
To do this it needs to be added to https://github.com/jetstack/cert-manager/tree/master/pkg/internal/apis/certmanager also so it will be converted

Reply:
OK. I tought the old api versions were frozen. I'll do that.

Reply:
They are for changes but not for new fields